.github/workflows/container-image.yml

```diff
@@ -18,7 +18,7 @@ jobs:
   docker:
     runs-on: ubuntu-latest
     steps:
-      # Allow multi-target builds
+      # Allow multi-target builds, uncomment this to use `platfotms` below
```

Comment:
When you say "uncomment this to use `platforms` below", what is "this" referring to?

Reply:
There is an element `platforms` further down the file.